pandas/core/nanops.py

```diff
@@ -633,10 +633,31 @@ def _kendall(a, b):
     def _spearman(a, b):
         return spearmanr(a, b)[0]
 
+    def _biweight_midcorrelation(a, b):
+        a_median = a.median()
+        b_median = b.median()
+
+        # Median absolute deviation
+        a_mad = (a - a_median).abs().median()
```

Comment:
@olgabot the python built-in `abs()` corresponds to the magic method `__abs__()` which pandas operator-overloads to call `np.abs()`, so it's fast 

for something like `sum()` though you'd totally be right, it's a lot slower to do `sum(series)`. You'd want to stick to `np.sum(series)` or `series.sum()`